app/assets/stylesheets/application.scss

```diff
@@ -30,6 +30,56 @@
  //   -ms-transform: translateY(-50%);
  //   transform: translateY(-50%);
  // }
+ .dev-tag{
+  color: black;
+  width: 30px;
+  background-color: #A80874;
+  margin: 5px 5px 5px 5px;
+  border-width: 1px;
+  border-style: solid;
+  border-color: black;
+ }
+
+ .test-tag{
+  color: black;
+  width: 32px;
+  background-color: #B7FDFE;
+  margin: 5px 5px 5px 5px;
+  border-width: 1px;
+  border-style: solid;
+  border-color: black;
+ }
+
+ .qa-tag{
+  color: black;
+  width: 30px;
+  background-color: #2B9720;
+  margin: 5px 5px 5px 5px;
+  border-width: 1px;
+  border-style: solid;
+  border-color: black;
+ }
+
+ .release-tag{
```

Comment:
Line should be indented 0 spaces, but was indented 1 spaces<br>Opening curly brace `{` should be preceded by one space